src/types/common/contactPoint.ts

```diff
@@ -25,55 +22,58 @@ export interface ContactPoint {
   use: ContactPointUse;
 }
 
-export const contactPointSchema = () =>
-  z.discriminatedUnion("system", [
+export const contactPoint = (t: (key: string) => string) => ({
```

Comment:
let's not pass `t` callback as an argument; let's keep it as how it was before itself!